packages/@okta/vuepress-site/docs/concepts/event-hooks/index.md

```diff
@@ -31,6 +31,12 @@ For general information on how Okta encapsulates events, see the [System Log API
 
 Event types include user lifecycle changes, the completion by a user of a specific stage in an Okta process flow, and changes in Okta objects. You can configure an event hook, for example, to deliver notifications of user deactivation events. You can use hooks to trigger processes that you need to execute internally every time a user is deactivated. For example, updating a record in an HR system, creating a ticket in a support system, or generating an email message.
 
+<ApiLifecycle access="ea" />
+
+You can reduce the number of event hook calls triggered by defining filters on specific instances of the subscribed event type. For example, if you want an event hook call triggered by user sign-in events for a specific group of users, you can filter on that group, rather than having an event hook call trigger for every user sign in. See [Create an event hook filter](#create-an-event-hook-filter).
```

Comment:
I find the first sentence a bit confusing as written. Does this still make sense?
```suggestion
You can reduce the number of triggered event hook calls by defining filters on specific instances of the subscribed event type. For example, if you want an event hook call triggered by user sign-in events for a specific group of users, you can filter on that group, rather than having an event hook call triggered for every user sign in. See [Create an event hook filter](#create-an-event-hook-filter).
```

Reply:
Good catch. It was a revision from a previous review. Since a call only occurs when it's triggered, it's really unnecessary in the sentence. The content stands by simply removing "triggered" in both those callouts. I'll do that. Thanks.